examples/asr/experimental/wav2vec/configs/wav2vec_pretrain.yaml

```diff
@@ -0,0 +1,122 @@
+name: &name "Wav2vec_Pretrain"
+
+model:
+  wav2vec:
```

Comment:
Is wav2vec level necessary. In other models, we used the model level for the configs of the model. If it is OK, let's make it consistent with others.